library/Garp/Cli/Command/Cache.php

```diff
@@ -28,8 +28,11 @@ public function clear(array $args = array()) {
             $cacheDir = $cache->getBackend()->getOption('public_dir');
         }
 
-        Garp_Cache_Manager::purge($args, true, $cacheDir);
-        Garp_Cli::lineOut('All cache purged.');
+        Garp_Cli::lineOut('Initialise cache clear');
+        $messageBag = Garp_Cache_Manager::purge($args, true, $cacheDir);
+        foreach ($messageBag as $message) {
+            Garp_Cli::lineOut($message);
+        }
```

Comment:
Je zou ook `Garp_Cli::lineOut(implode("\n", $messageBag))` kunnen doen. 

Dit is niet echt een waardeoordeel, gewoon een andere manier 🙂